app.py

```diff
@@ -17,9 +27,12 @@ def deployments():
     return render_template("deployments.html")
 
 
-@app.route("/open-pr")
+@app.route("/open_pr")
 def open_pr():
-    return render_template("open_pr.html")
+    reload_data = True if request.args.get("reload_data") == "true" else False
```

Comment:
If you use 1/0 as true/false for the reload_data argument in the URL, you can get rid of this logic and use `bool(request.args.get("reload_data"))`.

Reply:
done